plugins/html-editor/README.md

```diff
@@ -0,0 +1,89 @@
+# Builder.io example plugin
```

Comment:
readme needs a few instructions on how to enable and what this plugin do, and how to use it